docs/rpc_nodes_integration_test.md

```diff
@@ -9,7 +9,7 @@ author: Roxane Letourneau
 Remove `./rpc-nodes.spec.ts` from `"testPathIgnorePatterns"` in the package.json.
 2. In the integration test folder, run one of the following commands: 
     - **Hanghzounet**: `npm run test:hangzhounet rpc-nodes.spec.ts`
-    - **Kathmandunet**: `npm run test:kathmandunet rpc-nodes.spec.ts`
+    - **ghostnet**: `npm run test:ghostnet rpc-nodes.spec.ts`
     - **Both protocols**: `npm run test rpc-nodes.spec.ts`
```

Comment:
These docs are actually out of date, let's remove them or update them with correct values

Reply:
Do you mean the whole page? or just these instructions? the instructions get updated. ARe they entirely wrong?